scripts/build1.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+# this script builds a go binary after applying openssl fips patches.
+
+set -x
+export GOLANG_VER=1.22.2
+TOP_DIR=/tmp/golang-fips/build1
+[[ -d $TOP_DIR ]] || mkdir -p $TOP_DIR
+cd $TOP_DIR
+[[ -d golang-fips ]] || git clone https://github.com/golang-fips/go.git golang-fips
```

Comment:
This seems wrong, why would we download another copy? If you're running this script you already have this repo.

Reply:
True, I thought of users copy pasting it before cloning this repo. But that wont make much of a use case. Let em update this.

Reply:
Fixed.